aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -617,6 +647,15 @@ Tensor randn_like(
   return result.normal_(0, 1, c10::nullopt);
```

Comment:
reuse new `randn_like`:
```suggestion
  return native::randn_like(self, c10::nullopt, options, optional_memory_format);
```